sources/packages/web/src/views/institution/designations/DesignationView.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="p-m-4">
+    <header-navigator
+      title="Manage designations"
+      subTitle="View designations"
+      :routeLocation="{
+        name: InstitutionRoutesConst.MANAGE_DESIGNATION,
+      }"
+    />
+    <full-page-container class="mt-4">
+      <designation-agreement-form
+        :model="designationModel"
+      ></designation-agreement-form>
+    </full-page-container>
+  </div>
+</template>
+
+<script lang="ts">
+import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
+import { onMounted, reactive } from "vue";
+import {
+  useFormatters,
+  useInstitutionState,
+  useDesignationAgreement,
+} from "@/composables";
+import DesignationAgreementForm from "@/components/partial-view/DesignationAgreement/DesignationAgreementForm.vue";
+import {
+  DesignationModel,
+  DesignationFormViewModes,
+} from "@/components/partial-view/DesignationAgreement/DesignationAgreementForm.models";
+import { DesignationAgreementService } from "@/services/DesignationAgreementService";
+import HeaderNavigator from "@/components/generic/HeaderNavigator.vue";
+import { InstitutionRoutesConst } from "@/constants/routes/RouteConstants";
+
+export default {
+  components: { FullPageContainer, DesignationAgreementForm, HeaderNavigator },
+  props: {
+    designationAgreementId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const { institutionState } = useInstitutionState();
+    const formatter = useFormatters();
+    const { mapDesignationChipStatus } = useDesignationAgreement();
+
+    const designationModel = reactive({} as DesignationModel);
+    designationModel.institutionName = institutionState.value.operatingName;
+    designationModel.institutionType = institutionState.value.institutionType;
```

Comment:
if any of the store values like `operatingName` is changed, then will it reflect in `designationModel`. how about using a computed field?